Add multi-stage scenarios in Access Review API beta (#15387)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/accessreviewset-post-definitions.md

| instanceEnumerationScope | [accessReviewScope](../resources/accessreviewscope.md) | In the case of an all groups review, this determines the scope of which groups will be reviewed. See [accessReviewScope](../resources/accessreviewscope.md) and also learn how to [configure the scope of your access review definition](/graph/accessreviews-scope-concept).| 

> | reviewers | [accessReviewReviewerScope](../resources/accessreviewreviewerscope.md) collection | Defines who the reviewers are. If none are specified, the review is a self-review (users review their own access).  For examples of options for assigning reviewers, see [Assign reviewers to your access review definition using the Microsoft Graph API](/graph/accessreviews-reviewers-concept). |

Reviewer: ```suggestion
| reviewers | [accessReviewReviewerScope](../resources/accessreviewreviewerscope.md) collection | Defines who the reviewers are. If none are specified, the review is a self-review (users review their own access).  For examples of options for assigning reviewers, see [Assign reviewers to your access review definition using the Microsoft Graph API](/graph/accessreviews-reviewers-concept). If stageSettings are specified, the contents of this property will be ignored. |
```

Reviewer: @FaithOmbongi stageSettings has a reviewers property that overwrites this reviewers property. do you have any suggestions for informing readers of this info?

| scope | [accessReviewScope](../resources/accessreviewscope.md) | Defines the entities whose access is reviewed. See  [accessReviewScope](../resources/accessreviewscope.md) and also learn how to [configure the scope of your access review definition](/graph/accessreviews-scope-concept). Required.| 

> |stageSettings|[accessReviewStageSettings](../resources/accessreviewstagesettings.md) collection|The stage settings of an access review series. Stages will be created sequentially based on the dependsOn property. Each stage can have different set of reviewer, fallback reviewers and settings. See [accessReviewStageSettings](../resources/accessreviewstagesettings.md). Optional.|

Reviewer: ```suggestion
|stageSettings|[accessReviewStageSettings](../resources/accessreviewstagesettings.md) collection| Defines how many stages each instance of an access review series will have. Stages will be created sequentially based on the dependsOn property. Each stage can have different set of reviewer, fallback reviewers and settings. See [accessReviewStageSettings](../resources/accessreviewstagesettings.md). Optional.|
```

}
```

> ### Example 4: Create an access review on a group with multi-stage

Reviewer: ```suggestion
### Example 4: Create an access review on a group with multiple stages
```
